lib/services/ios-debug-service.ts

```diff
@@ -212,7 +213,7 @@ export class IOSDebugService extends DebugServiceBase implements IPlatformDebugS
 				this.$logger.info("'--chrome' is the default behavior. Use --inspector to debug iOS applications using the Safari Web Inspector.");
 			}
 
-			this._socketProxy = await this.$socketProxyFactory.createWebSocketProxy(this.getSocketFactory(device));
+			this._socketProxy = await this.$net.getAvailablePortInRange(41000);
```

Comment:
This bypasses a lot of logic, please check it.

Reply:
Right, must have pushed the wrong thing